gateway/1.9/modules/ROOT/pages/flex-managed-configure.adoc

```diff
@@ -0,0 +1,8 @@
+= Configuring Managed Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:page-mode: conn
+
+* xref:flex-managed-tls-config.adoc[] - Managed Flex Gateway on CloudHub 2.0 enables you to configure a TLS context to support regular TLS and mTLS in both inbound and outbound directions.
```

Comment:
All tls inbound configuration need to be done in Cloudhub for Managed Flex Gatewayhttps://docs.mulesoft.com/cloudhub-2/ps-config-domains